### PR TITLE
Refactor common Airdrop Utils for Login Flow

### PR DESCRIPTION
**Pull request by pankaj-gkm**
- Changed base url path in next config
- Refactored code
- Add util function to handle cookiess

### CODE REVIEW

File: modules/Authentication/lib/constants.ts [hunk 1]
```diff
@@ -109,3 +110,9 @@ export const appRoutes = {
   TRANSACTION_HISTORY: '/transaction-history/',
   VIDEO_DETAIL: '/video-details/[videodetail]',
 };
+
+export const LOGIN_METHOD = {
+  phoneNumber: 'phone_number',
+  google: 'google',
+  discord: 'discord',
```
Comment (shelly19gkmit) [inline modules/Authentication/lib/constants.ts:117]: For otpless?

File: modules/Authentication/utils/index.ts [hunk 2]
```diff
@@ -1,6 +1,11 @@
 // import { SxProps } from "@mui/material";
 // import { INDIAN_DELETE_CLAN_FLOW, INDIAN_LOGIN_FLOW } from "../kgen/indianLoginFlow";
-import {Flows} from '../lib';
+import {
+  Flows,
+  LOGIN_METHOD,
+  VerifyOtpApiResponse,
+  local_Storage_Keys,
+} from '../lib';
```
Comment (shelly19gkmit) [inline modules/Authentication/utils/index.ts:8]: alias for lib?

File: modules/Authentication/lib/constants.ts (hunk 1 above)
Comment (pankaj-gkm) [reply modules/Authentication/lib/constants.ts:117]: These were the only LOGIN_METHODS on kgen Repo, does other need to added? whatsapp and twitter?

Comment (shelly19gkmit) [reply modules/Authentication/lib/constants.ts:117]: yes

File: modules/Authentication/hooks/login/loginFlow/useUpdateLoginFlowAtom.ts [hunk 3]
```diff
@@ -1,12 +1,19 @@
+import {useEffect} from 'react';
 import {useAtom} from 'jotai';
 import {useMonitorAtomState} from '@auth-hooks';
 import {loginFlowAtom} from '@auth-store';
 import {LoginFlowType} from '@auth-lib';
+import {getRecievedToken} from '@auth/utils';
```
Comment (shelly19gkmit) [inline modules/Authentication/hooks/login/loginFlow/useUpdateLoginFlowAtom.ts:6]: Why 2 different types of imports?
@auth-lib and @auth/utils

Comment (pankaj-gkm) [reply modules/Authentication/hooks/login/loginFlow/useUpdateLoginFlowAtom.ts:6]: There are two utils file
modules/lib/utils
modules/utils

@auth-utils is for modules/utils

**Review by shelly19gkmit — APPROVED** [review]

**Pull request merged**